scripts/GameBoard.class.js

```diff
@@ -11,28 +11,19 @@ let blocksOnPage,
     currentSpeed,
     elementsOnBoard,
     intervalID,
-    gameFinishedFlag,
-    numberOfBlocks;
-
-function checkInputValue() {
-    let value = +document.querySelector('#number').value;
-
-    return value >= 9 && value <= 15 ? value : 9;
-}
+    gameFinishedFlag;
 
 function setInitValues() {
```

Comment:
Why this method is outside of constructor function? I suppose, it should be a part of class, no?